.github/workflows/build.yaml

```diff
@@ -0,0 +1,59 @@
+#
+name: Create and publish a Docker image
+
+# Configures this workflow to run every time a change is pushed to the branch called `release`.
+on:
+    push:
+        branches: ["main"]
```

Comment:
Can you add branch `container` to test the workflow and then remove it before merging?

And also add release or tag here as well? And then you can do a release or tag on this branch to test the image being properly tagged.

Reply:
Done